### PR TITLE
misc: remove unnecessary injects for title opacity

### PR DESCRIPTION
**Pull request by Wyvest**

### CODE REVIEW

File: src/main/java/club/sk1er/patcher/mixins/features/GuiIngameForgeMixin_TitleRendering.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public GuiIngameForgeMixin_TitleRendering(Minecraft mc) {
     private void patcher$modifyTitle(int l, int age, float opacity, CallbackInfo ci) {
         float titleScale = PatcherConfig.titleScale;
         if (PatcherConfig.autoTitleScale) {
-            float width = fontrenderer.getStringWidth(displayedTitle) * 4.0F;
+            final float width = fontrenderer.getStringWidth(displayedTitle) * 4.0F;
```
Comment (DJtheRedstoner) [inline src/main/java/club/sk1er/patcher/mixins/features/GuiIngameForgeMixin_TitleRendering.java:37]: Unneeded change.

File: src/main/java/club/sk1er/patcher/mixins/features/GuiIngameForgeMixin_TitleRendering.java [hunk 2]
```diff
@@ -45,21 +45,16 @@ public GuiIngameForgeMixin_TitleRendering(Minecraft mc) {
     private void patcher$modifySubtitle(int l, int age, float opacity, CallbackInfo ci) {
         float titleScale = PatcherConfig.titleScale;
         if (PatcherConfig.autoTitleScale) {
-            float width = fontrenderer.getStringWidth(displayedSubTitle) * 2.0F;
+            final float width = fontrenderer.getStringWidth(displayedSubTitle) * 2.0F;
```
Comment (DJtheRedstoner) [inline src/main/java/club/sk1er/patcher/mixins/features/GuiIngameForgeMixin_TitleRendering.java:48]: See above.

**Review by DJtheRedstoner — COMMENTED** [review]
What's the motivation here? As far as I can tell, this isn't the same behavior. When a title fades out, the fade will now be "cut off" by the clamp instead of fading from the set opacity to 0.

**Comment by Wyvest** [comment]
> What's the motivation here? As far as I can tell, this isn't the same behavior. When a title fades out, the fade will now be "cut off" by the clamp instead of fading from the set opacity to 0.

won't doing this make it always less than the set opacity via clamp

**Comment by DJtheRedstoner** [comment]
> won't doing this make it always less than the set opacity via clamp
wdym? To be clear about what I mean: before, if the opacity was half, the title would fade from 128 -> 0 alpha. Now, it would fade from 255 -> 0, but be clamped at a max of 128 (effectively halving the duration of the animation).

**Pull request closed without merging**